### ISSUE TITLE
yaml parser: short notation for operation assignment not working

### ISSUE THREAD

**Issue by hnicke** [eclipse/winery#215]
Encountered a bug in the yaml parser: He doesn't handle short notation for operation assignment.

#### Minimal yaml example
When feeding the tosca yaml parser with:

```yaml
tosca_definitions_version: tosca_simple_yaml_1_1

description: a template for testing handling of the short notation for an operation assignment
metadata:
  template_name: short-notation_operation-assignment
  template_author: stupro-toscana
  template_version: 1.0

topology_template:
  node_templates:
    ny_node:
      type: tosca.nodes.Database
      interfaces:
        Standard:
          configure: script.sh          
```

#### Expected behaviour:
The resulting TInterfaceDefintion's operation map contains the configure operation.
See TOSCA yaml Simple Profile Specification v1.1 cos01 chapter 3.5.13.2.1
> The following single-line grammar may be used when only an operation’s implementation artifact is
needed:
><operation_name>: <implementation_artifact_name>

#### Witnessed behaviour:
The resulting TInterfaceDefinition's operation map is empty.

@kleinech (merry christmas, anyways)

**Comment by kleinech** [comment]
Currently only the first option from TOSCA YAML Simple Profile Specification v1.1 cos01 chapter 3.5.13.2.3
>implementation_artifact_name: represents the optional name (string) of an implementation
artifact definition (defined elsewhere), or the direct name of an implementation artifact’s relative
filename (e.g., a service template-relative, path-inclusive filename or absolute file location using a
URL).

is supported.

**Comment by koppor** [comment]
Fixed via #218.
